rio/src/RIO/Prelude/Logger.hs

```diff
@@ -292,11 +292,19 @@ logOptionsMemory = do
 -- to perform verbose logging or not. Individiual settings can be
 -- overridden using appropriate @set@ functions.
 --
+-- When Debug Flag is @True@:
+--
+--     * enables @logVerboseFormat@ option
+--     * enables @logUseColor@ when handle is a terminal
+--     * enables @logUseLoc@ option
+--     * enables @logUseTime@ option
+--     * sets @logMinLevel@ to 'Debug'
+--
 -- @since 0.0.0.0
 logOptionsHandle
   :: MonadIO m
   => Handle
-  -> Bool -- ^ verbose?
+  -> Bool -- ^ Debug Flag:
```

Comment:
I don't see it as better to call this "debug flag": that implies that the only difference is whether it's showing Debug logging.